### ISSUE TITLE
The CholecT5 is the CholecT50-Challenge-Validation？ 

### ISSUE THREAD

**Issue by csEylLee** [CAMMA-public/cholect50#2]
The CholecT5 is the CholecT50-Challenge-Validation？

**Comment by nwoyecid** [comment]
CholecT50 is the complete dataset.
CholecT45 is the first 45 videos of the CholecT50 and also the Challenge training.
CholecT50-Challenge-validation set is a 5 short clip dataset (not complete videos like others) sampled from CholecT50 training set.
